Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.py

```diff
@@ -0,0 +1,1169 @@
+        outputs=res,
+        readable_output=readable_output,
+        raw_response=res
+    )
+
+
+def find_processes_results_command(client: Client, args: dict):
+    job_id = args.get('job_id')
+    rows = args.get('rows')
+
+    if not job_id:
+        return CommandResults(
+            readable_output="The job id can't be empty",
+            raw_response="The job id can't be empty"
+        )
+
+    if not rows:
+        rows = 10
+    res = client.get_process_results(job_id, rows)
+    headers = ['device_id', 'device_name', 'process_name', 'device_policy_id', 'enriched_event_type']
+
+    human_readable = res.get('results')
+    readable_output = tableToMarkdown('The Results For The Process Search',
+                                      human_readable,
+                                      headers=headers,
+                                      removeNull=True,
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefense.Process.Results',
+        outputs_key_field='id',
+        outputs=res,
+        readable_output=readable_output,
+        raw_response=res
+    )
+
+
+def alerts_search_command(client: Client, args: dict):
+    alert_type = args.get('type', 'all')
+    category = argToList(args.get('category'))
+    device_id = argToList(args.get('device_id'))
+    first_event_time = args.get('first_event_time')
+    policy_id = argToList(args.get('policy_id'))
+    process_sha256 = argToList(args.get('process_sha256'))
+    reputation = argToList(args.get('reputation'))
+    tag = argToList(args.get('tag'))
+    device_username = argToList(args.get('device_username'))
+    query = args.get('query')
+    rows = args.get('rows')
+    start = args.get('start')
+    headers = ['id', 'category', 'device_id', 'device_name', 'device_username', 'create_time', 'ioc_hit', 'policy_name',
+               'process_name', 'type', 'severity']
+    human_readable = []
+
+    if first_event_time:
+        first_event_time = json.loads(first_event_time)
+    res = client.get_alerts(alert_type, category, device_id, first_event_time, policy_id, process_sha256, reputation,
+                            tag, device_username, query, rows, start)
+
+    alerts = res.get('results', [])
+    if not alerts:
+        return 'No alerts were found.'
+    for alert in alerts:
+        human_readable.append({
+            'id': alert.get('id'),
+            'category': alert.get('category'),
+            'device_id': alert.get('device_id'),
+            'device_name': alert.get('device_name'),
+            'device_username': alert.get('device_username'),
+            'create_time': alert.get('create_time'),
+            'ioc_hit': alert.get('ioc_hit'),
+            'policy_name': alert.get('policy_name'),
+            'process_name': alert.get('process_name'),
+            'type': alert.get('type'),
+            'severity': alert.get('severity')
+        })
```

Comment:
Why are we editing the result? Why not just the specific headers to tableToMarkdown?